src/main/java/com/attacktimer/AttackTimerMetronomeConfig.java

```diff
@@ -55,24 +57,21 @@ default boolean disableFontScaling()
 		return false;
 	}
 
-	@Range(
-			min = 8,
-			max = 50
-	)
 	@ConfigItem(
-			position = 4,
+			position = 3,
 			keyName = "fontSize",
 			name = "Font Size (Overhead Tick Only)",
 			description = "Change the font size of the overhead attack cooldown ticks",
 			section = TickNumberSettings
 	)
+	@Range(min = 8, max = 50)
 	default int fontSize()
 	{
-		return 15;
```

Comment:
Same comment about keeping the same defaults.

Reply:
Yeah, I can agree with this one, it's just too small on some screens so I decided to set it a little higher ahah, I will revert this